src/custom/api/gnosisProtocol/api.ts

```diff
@@ -274,12 +277,23 @@ const UNHANDLED_ORDER_ERROR: ApiErrorObject = {
   description: ApiErrorCodeDetails.UNHANDLED_CREATE_ERROR,
 }
 
-async function _handleQuoteResponse<T = any, P extends QuoteQuery = QuoteQuery>(
+async function _handleQuoteResponse<T = any, P extends MinimumSentryQuoteData = MinimumSentryQuoteData>(
   response: Response,
   params?: P
 ): Promise<T> {
   if (!response.ok) {
-    const errorObj: ApiErrorObject = await response.json()
+    const errorObj: ApiErrorObject = await response.json().catch((error) => {
```

Comment:
I would avoid parsing the JSON if the content type is not JSON. 

Can u check if the backend sends the header, I would assume so. So you can just do, if it's JSON u parse it, if not, u don't :)